store/key_value_store_server.cc

```diff
@@ -35,12 +34,57 @@ bool KvstoreServiceImpl::fileExists(const char *fileName)
 }
 
 void KvstoreServiceImpl::CreateKeyFile(const std::string &key, const std::string &value){
+  if (!fileExists("key-pair.txt")){
+    std::ofstream myfile;
+    myfile.open("key-pair.txt", std::ios_base::app);
+    myfile << key << ":" << value << "\n";
+    myfile.close();
+  } else {
+    std::fstream infile("key-pair.txt");
+    std::string line;
+    std::string delimiter = ":";
+    int i = 0;
+    const char * v = value.c_str();
+    const char * k = key.c_str();
```

Comment:
It may be fine to just use strings here.

Reply:
For now the libraries Im using are taking in these rather than std::strings.. maybe my mistake will try to change it around